### ISSUE TITLE
Wrong Number in the DMG Enumerations List

### ISSUE THREAD

**Issue by Tr4ckster** [Facepunch/garrysmod-issues#3497]
### Details
When printing out the Damagetype as number, used in the https://wiki.garrysmod.com/page/Enums/DMG
the DMG_BUCKSHOT Number in the list is 536870912, but in gmod printed it is 536870914. Means if you use the Buckshot in a if statement for example the number of the list in the wiki would never be true even if you get shot by a shotgun. Making stuff not work, and the number in the list being wrong.
I figured it out because i create a addon that reacts to different damage types.

### Steps to reproduce

If you execute this Script
`hook.Add("EntityTakeDamage", "PlayerTookDamage", function(ply , dmginfo)
	local dmgtype = dmginfo:GetDamageType()
	print(dmgtype)
end)`

print will give 536870914, while the gmod wiki list says it is 536870912. (the last digit it 4 instead of 2)
Basically the DMG Enumerations List needs a Update

**Comment by yupi2** [comment]
Binary OR'd with DMG_BULLET (2) probably. You should use `if bit.band(dmgtype, DMG_BUCKSHOT) then`
